Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.cpp

```diff
@@ -404,6 +410,8 @@ namespace AZ
 
         void CommandList::Submit(const RHI::DrawItem& drawItem)
         {
+            ValidateSubmitItem(drawItem);
```

Comment:
Just a note for the future, might be good to have a common function that all the backend implementations call where we can put common functionality like the submit item validation.